ArmPkg/Library/SecurePartitionEntryPoint/StandaloneMmCoreEntryPoint.c

```diff
@@ -423,12 +422,12 @@ ModuleEntryPoint (
   // Update the global copy now that the image has been relocated.
   mUseOnlyFfaAbis = UseOnlyFfaAbis;
 
-  ProcessLibraryConstructorList (NULL, NULL);
+  ProcessLibraryConstructorList (DtbAddress, NULL);

```

Comment:
I have concerns about how ImageHandle is being taken to pass DtbAddress.

Taking ImageHandle and using it in this manor seems like we are going to cause more problems down the line when we encounter a library constructor that attempts to use the passed values as they are stated in the prototypes.

Reply:
Yeah. I was thinking about the same. The hard part is that this FDT address will be automatically populated by the loader in the input parameters at the entrypoint function. We could save it to a global variable and have others access it that way. But since I imagine this will be used in other libraries as well, using an extern will be a bad pattern. Any suggestion is welcome.

Reply:
summarizing our offline conversation:

UefiBootServicesTableLib and UefiRuntimeServicesTableLib create gBS and gRT global variables, and their implementations are provided by the library instance.
Would doing something similar for DtbAddress be acceptable?

Reply:
Changes look good to me.